sdk/src/main/java/com/bugsnag/android/SessionTrackingPayload.java

```diff
@@ -49,4 +49,8 @@ public void toStream(@NonNull JsonStream writer) throws IOException {
     Session getSession() {
         return session;
     }
+
+    Map<String, Object> getDevice() {
```

Comment:
Nit pick, but I don't think the change to this file belongs in this PR.

Reply:
I probably should have mentioned that there have been some changes in this [react native PR](https://github.com/bugsnag/bugsnag-react-native/pull/345) which mean that this accessor is required in order to alter the payload.